packages/bento-design-system/src/Table/cells.tsx

```diff
@@ -30,12 +30,17 @@ export function ButtonCell({
 }: CellProps<{}, Omit<ButtonProps, "size">> & {
   options: Partial<Pick<ButtonProps, "size">>;
 }) {
+  const tableConfig = useBentoConfig().table;
+  const buttonConfig = useBentoConfig().button;
+  const buttonSize = size ?? "medium";
   return (
-    <Inset space={8}>
-      <Inline space={0} align={align} alignY="center">
-        <Button size={size ?? "medium"} {...buttonProps} />
-      </Inline>
-    </Inset>
+    <Bleed spaceX={buttonConfig.paddingX[buttonSize]} spaceY={buttonConfig.paddingY[buttonSize]}>
```

Comment:
I had to refactor that, because, talking with @Phyele, we came to the conclusion that having those two separate configurations interact in this unintuitive way wasn't nice. Have you had a look at the updated version?